server/models/services/AtService.js

```diff
@@ -255,17 +276,17 @@ const getAtVersions = async (
  * @returns {Promise<*>}
  */
 const createAtVersion = async (
-    { atId, atVersion },
+    { atId, name, releasedAt = null },
```

Comment:
Maybe releasedAt doesn't need a default value? Because I think it's a required field.